board/piksiv3/post_build.sh

```diff
@@ -23,3 +23,7 @@ else
   echo "*** NO FIRMWARE FILES FOUND, SKIPPING ***"
 fi
 
+# Restore monit config file permissions that aren't preserved by git
+# Monit will refuse to work without these permissions set
+chmod 600 $ROOTFS/etc/monitrc
```

Comment:
I think this should be accomplished using the device table: https://github.com/swift-nav/piksi_buildroot/blob/master/board/piksiv3/device_table.txt. I don't believe there is any guarantee that target permissions will match the host.